src/Bicep.Core.IntegrationTests/CompileTimeImportTests.cs

```diff
@@ -38,15 +38,17 @@ public void Importing_unexported_symbol_should_raise_diagnostic()
     {
         var result = CompilationHelper.Compile(ServicesWithCompileTimeTypeImports,
             ("main.bicep", """
-                import {foo} from 'mod.bicep'
+                import {foo, bar} from 'mod.bicep'
```

Comment:
Could you add some baselines for the `import` feature generally? I might be missing them but couldn't find any samples searching the repo for files with name `*.bicep`